### ISSUE TITLE
Feature request: support for restoring Ndarrays from compressed blobs

### ISSUE THREAD

**Issue by rnauber** [Blosc/python-blosc2#114]
Hi,
first of all, thanks for developing this great library, I am using it daily in the context of scientific computing!
However, I want to use the new `Ndarray` for transmitting numpy data over the wire. To serialize the data, I can use this: 

```
blosc_ndarray = blosc2.asarray(ndarray, cparams=cparams)
compressed_blob = blosc_ndarray.schunk.to_cframe()
```

However, I have not found a way to get an `ndarray` back from the `compressed_blob` (except for writing it to a file).  Would it be possible to support this use case?

Thank you very much!
Best,
Richard

**Comment by FrancescAlted** [comment]
Implemented in #147

**Comment by rnauber** [comment]
Thanks a lot, @FrancescAlted !
